server/cmd/genkey/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v2"
+)
+
+func actualMain() error {
+	secPrivKey, err := secp256k1.GeneratePrivateKey()
+	if err != nil {
+		return err
+	}
+	// privkeyFile, err := os.OpenFile("dexprivkey.pem", os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
+	// if err != nil {
+	// 	return err
+	// }
+	// defer privkeyFile.Close()
+
+	// err = pem.Encode(privkeyFile, &pem.Block{
+	// 	Type:  "EC PRIVATE KEY",
+	// 	Bytes: secPrivKey.Serialize(),
+	// })
+	// if err != nil {
+	// 	return err
+	// }
+
+	err = ioutil.WriteFile("dexprivkey", secPrivKey.Serialize(), 0600)
+	if err != nil {
+		return err
+	}
```

Comment:
It would be nice if the openssl command could be used to generate the secp256k1 keys in a format the dcrdex can read.  We can read PEM, but the way openssl encodes the curve parameters and the public key in the same element is not clear to me yet.  We would also want to write in a similar way so openssl could inspect the keys.  This will take more work and it's not a show stopper, so I've just fallen back to unstructured binary files.

Reply:
cspp does this so I will look into the approach there and follow up on this.